.gitattributes

```diff
@@ -4,60 +4,6 @@
 * text=auto
 
```

Comment:
you'll need to change the yml file in Utilities\Pipelines\Tasks. The pipeline calls the bash script and you changed their location

Reply:
fixed